.gitmodules

```diff
@@ -8,9 +8,9 @@
 [submodule "extern/test-vectors"]
 	path = extern/test-vectors
 	url = https://github.com/filecoin-project/test-vectors.git
-[submodule "extern/fil-blst"]
-	path = extern/fil-blst
-	url = https://github.com/filecoin-project/fil-blst.git
 [submodule "extern/oni"]
 	path = extern/oni
 	url = https://github.com/filecoin-project/oni
+[submodule "extern/blst"]
+	path = extern/blst
+	url = git@github.com:supranational/blst.git
```

Comment:
This breaks the build on machines that don't have github pubkeys setup. You need https here, not git.